Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticFilters.tsx

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import React, { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { FontSizes, SharedColors, NeutralColors } from '@uifabric/fluent-theme';
+import { DiagnosticSeverity } from '@bfc/shared';
+import { FontWeights } from '@uifabric/styling';
+
+import { outputsDebugPanelSelector, rootBotProjectIdSelector } from '../../../../../recoilModel';
+import { DropdownWithAllOption } from '../../../../../components/DropdownWithAllOption/DropdownWithAllOption';
+
+const severityTextStyle = {
+  marginRight: '6px',
+};
+
+const countFontWeight = {
+  fontWeight: FontWeights.bold,
+};
+
+const optionAllKey = 'All';
+
+const getSeverityButtonStyle = (severityType: DiagnosticSeverity, isChecked: boolean): any => {
+  const baseStyle = {
+    root: {
+      padding: '8px 4px',
+      marginRight: '8px',
+      fontSize: FontSizes.size12,
+      fontFamily: 'Segoe UI',
```

Comment:
is this needed?

Reply:
Nope. Not required. cleaning up this from the old code.